docs/logs/field-types.md

```diff
@@ -0,0 +1,106 @@
+You can see field types that you can chose in Field Editor when adding or modifying a field.
```

Comment:
Note that we use Title: <title> as the very first line, followed by an empty line to tell Mkdocs to use that as the page heading and also in <title> tag.

Reply:
I thought that inheriting the title from the file name ("Field types") is enough. Should I explicitly write "title: Field types". Is this what you mean?

Reply:
Does inheriting make Mkdocs create a <title>Field types</title>?  If not, then explicit Title: Field types as the very first line is needed.

Reply:
This is what it looks like. I can try to type "title ..." explicitly to see how it behaves

![selection_376](https://user-images.githubusercontent.com/6692291/39998092-4434f9ee-5785-11e8-8044-00c0743826f4.png)

Reply:
I tried it and with "title: ..." it looks the same. Do you think that makes sense to duplicate the name and put it in 2 places (menu and title explicitly inside the file)? I am afraid that we may forget to change one place and I guess in 99% cases we need the same title in the menu and inside the file.

Reply:
Key thing: just because you see a *heading* on a page doesn't mean that there is actually <title> tag in the page :) . So that's what you want to look for.

Reply:
I thought you were referring to the heading but there is also `<title>` tag  in HTML if this is what you mean

Reply:
For future reference.  There always is a <title> tag in the generated HTML.  The question is just what Mkdocs put in it.  Think Google SERPs, think SEO.  It used to put some default until we figured out we could use the Title: "trick" to tell it which exact <title>....</title> to use.

Reply:
Mkdocs put there a title of the menu element. I really checked this.